azure-kusto-data/tests/test_telemetry.py

```diff
@@ -11,29 +11,70 @@ class TestTelemetry:
 
     @staticmethod
     def plus_one(num):
-        KustoTracingAttributes.add_attributes(tracing_attributes={"foo": "bar"})
+        SpanAttributes.add_attributes(tracing_attributes={"foo": "bar"})
         return num + 1
 
     @staticmethod
     async def plus_one_async(num):
-        KustoTracingAttributes.add_attributes(tracing_attributes={"foo": "bar"})
+        SpanAttributes.add_attributes(tracing_attributes={"foo": "bar"})
         return num + 1
 
     @staticmethod
-    def test_call_func_tracing():
-        res = KustoTracing.call_func_tracing(TestTelemetry.plus_one, 1, name_of_span="plus_one")
-        assert res == 2
+    def test_run_valid_invoker():
+        # Happy path test for run method with valid invoker function and name of span
+        def invoker():
+            return "Hello World"
+
+        span = Span.run(invoker, "test_span")
+        assert span is not None
+
+    @staticmethod
+    async def test_run_async_valid_invoker():
```

Comment:
async tests need "@pytest.mark.asyncio" or they're not doing anything

Reply:
thanks